src/jmespath.net/jmespath.net.csproj

```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>JmesPath.Net is a fully compliant JMESPATH parser.</Description>
     <AssemblyTitle>JmesPath.Net</AssemblyTitle>
     <Authors>Jeremie Devillard;Maxime Labelle</Authors>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;net45</TargetFrameworks>

```

Comment:
Please, update `src/jmespath.net.signed/jmespath.net.signed.csproj` accordingly.

Reply:
Please, update the corresponding `tests/jmespathnet.tests/jmespathnet.tests.csproj` with an additionnal target framework defined to `netcoreapp1.0`.

Also, add a corresponding trigger for this test configuration in `appveyor.yml` by adding a line in the `test_script` section of the file.